public/classes/FriendlyCaptcha/Captcha.php

```diff
@@ -0,0 +1,153 @@
+<?php
```

Comment:
can we put them all in src/ and just keep puzzle.php and siteverify.php in public/?

Reply:
Sure. Do you prefer "src/classes/FriendlyCaptcha/Captcha.php" or "src/FriendlyCaptcha/Captcha.php"?

Reply:
without classes/ .... and while we are on this, we should add another sub-namespace.

FriendlyCaptcha/Lite/Captcha.php ...

Reply:
I moved the classes as requested